chord_metadata_service/restapi/authz_middleware.py

```diff
@@ -13,9 +13,14 @@ def __init__(self, get_response):
 
     def __call__(self, request):
         if request.path == "/api/phenopackets" and request.method == "GET":
```

Comment:
Minor question - if this path is the only path that should be forwarding the headers?  I am asking because in like 23 you pass the `request.path` in the body so maybe in the future we may have a list or paths?

Reply:
The reason that we only have the `/api/phenopackets` path is because right now this is the only path where the OPA service is called and hence why this path is the only one that forwards the headers. @ljdursi can correct me if I'm wrong but in the future this will expand to include more paths beyond `/api/phenopackets`.